test/integration/sequential-feed.spec.ts

```diff
@@ -39,10 +39,12 @@ describe('feed', () => {
   }, 21000)
 
   test('multiple updates using setUpdate and lookup', async () => {
-    const reference = Utils.Hex.makeHexString('0000000000000000000000000000000000000000000000000000000000000000', 64)
+    const reference = Utils.Hex.makeHexString(new Date().getTime().toString().padStart(64, '0'), 64)
```

Comment:
why did you touch this part?
1. it is not related to the streaming of feeds
2. it messes up the inteded test logic as you check the code below.